cmd/fluxsvc/main.go

```diff
@@ -99,21 +99,20 @@ func main() {
 		busMetrics       platform.BusMetrics
 		helperDuration   metrics.Histogram
 		historyMetrics   history.Metrics
-		httpDuration     metrics.Histogram
+		httpDuration     *stdprometheus.HistogramVec
 		instanceMetrics  instance.Metrics
 		jobWorkerMetrics jobs.WorkerMetrics
 		registryMetrics  registry.Metrics
 		releaseMetrics   release.Metrics
 		serverMetrics    server.Metrics
 	)
 	{
-		httpDuration = prometheus.NewHistogramFrom(stdprometheus.HistogramOpts{
+		httpDuration = stdprometheus.NewHistogramVec(stdprometheus.HistogramOpts{
```

Comment:
One thing this loses is the request _counts_ (and therefore rates). Does the standard instrumentation not care about those?